### PR TITLE
PORT-2479 Plenty of docs fixes and updates

### PR DESCRIPTION
**Pull request by MPTG94**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

**Review by danielsinai — APPROVED** [review]
🦢

File: docs/complete-use-cases/sdlc.md [hunk 1]
```diff
@@ -124,6 +124,20 @@ The Blueprint JSON provided below already includes the Relations between the dif
 }
 ```
 
+After creating the initial service Blueprint, go back and edit the Blueprint again, but replace the `relations` key with the following `relations` object:
```
Comment (EllaFurman) [inline docs/complete-use-cases/sdlc.md:127]:
```suggestion
After creating the initial service Blueprint, return to the Blueprint and replace the `relations` key with the following `relations` object:
```

File: docs/complete-use-cases/sdlc.md [hunk 2]
```diff
@@ -448,6 +462,19 @@ properties:
   helmLink: "https://github.com/port-labs/helm/notification.yml"
 ```
 
+:::info
+Recall that you created a Relation from the service Blueprint to itself, allowing you to map dependencies between your services. If you want to add a service that `notification-service` depends on, you can do that by adding the following `relations` key to the `port.yml` (some services were added as an example):
```
Comment (EllaFurman) [inline docs/complete-use-cases/sdlc.md:466]:
```suggestion
Before, you created a Relation from the service Blueprint to itself. That Relation allowed you to map dependencies between your services. If you want to add a service on which `notification-service` depends, add the following `relations` key to the `port.yml` (notice some services were added as an example):
```

File: docs/software-catalog/blueprint/blueprint.md [hunk 3]
```diff
@@ -69,15 +69,15 @@ Each Blueprint is represented by a [Json schema](https://json-schema.org/), as s
...
+| `calculationProperties` | `Object` | Contains the properties that are defined using [calculation templates](./calculation-properties)                                                  | Example: "`repo-link`": "`https://github.com/{{$identifier}}`"                                                                                |
+| `schema`                | `Object` | Object containing two more nested fields, including `properties` and `required`                                                                   | See the schema structure [here](#blueprint-schema).                                                                                           |
+| `changelogDestination`  | `Object` | Defines the destination where events that happen within the Blueprint's context will be delivered                                                 | See the object structure [here](#changelog-destination).                                                                                      |
```
Comment (EllaFurman) [inline docs/software-catalog/blueprint/blueprint.md:80]:
```suggestion
| `identifier`            | `String` | Unique identifier.<br /> Note that while the identifier is unique, it [can be changed](./tutorial.md/#update-blueprint-identifier) after creation |
| `title`                 | `String` | Blueprint's name.                                                                                                                             |
| `description`           | `String` | Description for the Blueprint.<br /> The value is visible to users when hovering over the info icon in the UI.                                     |
| `icon`                  | `String` | Icon for the Blueprint's graph node and Entities of the Blueprint.                                                                                | Icon options: `Airflow, Ansible, Argo, Aws, Azure, Blueprint, Bucket, Cloud,...` <br /><br />See the full icon list [below](#full-icon-list). |
| `calculationProperties` | `Object` | Contains the properties defined using [calculation templates](./calculation-properties).                                                  | Example: "`repo-link`": "`https://github.com/{{$identifier}}`"                                                                                |
| `schema`                | `Object` | An object containing two more nested fields, including `properties` and `required`.                                                                   | See the schema structure [here](#blueprint-schema).                                                                                           |
| `changelogDestination`  | `Object` | Defines the destination where events that happen within the Blueprint's context will be delivered                                                 | See the object structure [here](#changelog-destination).                                                                                      |
```

Comment (EllaFurman) [inline docs/software-catalog/blueprint/blueprint.md:80]: The last row of the table isn't clear at all

File: docs/software-catalog/blueprint/tutorial.md [hunk 4]
```diff
@@ -331,12 +331,35 @@ You can always modify Blueprints, add/remove properties or change existing prope
 
 To update a Blueprint, you can do either of the following:
 
-- Click the pencil icon in the Blueprints graph;
+- Click the 3 dots (`...`) button and then click the `Properties` button in the Blueprints graph;
```
Comment (EllaFurman) [inline docs/software-catalog/blueprint/tutorial.md:334]:
```suggestion
- In the Blueprints graph, click on the 3 dots (`...`) button and after it the `Properties` button;
```

File: docs/software-catalog/blueprint/tutorial.md [hunk 5]
```diff
@@ -331,12 +331,35 @@ You can always modify Blueprints, add/remove properties or change existing prope
...
 - Make an HTTP PUT request to the URL: `https://api.getport.io/v1/{blueprint_identifier}`
+  - A PUT request has the same body as a POST request, it will simply overwrite the Blueprint with the new data provided.
 
 ![Blueprints Graph edit button marked](../../../static/img/software-catalog/blueprint/blueprintGraphEditButtonMarked.png)
 
-A PUT request has the same body as a POST request, it will simply overwrite the Blueprint with the new data provided.
+### Update Blueprint identifier
+
+In order to change the identifier of a Blueprint you can use one of the following options:
```
Comment (EllaFurman) [inline docs/software-catalog/blueprint/tutorial.md:342]:
```suggestion
There are two methods to to change a Blueprint's identifier:
```

File: docs/software-catalog/blueprint/tutorial.md [hunk 6]
```diff
@@ -331,12 +331,35 @@ You can always modify Blueprints, add/remove properties or change existing prope
...
+
+#### From the UI
+
+1. In the Blueprints graph, on the Blueprint you want to update, click the 3 dots (`...`) button and then click the `Properties` button;
+2. Update the value of the `identifier` key to the new identifier;
+3. Click `Save`.
```
Comment (EllaFurman) [inline docs/software-catalog/blueprint/tutorial.md:348]:
```suggestion
1. In the desired Blueprints graph, click on the 3 dots (`...`) button and then on the `Properties` button;
2. Update the value of the `identifier` key to the new identifier;
3. Click `Save`.
```

File: docs/software-catalog/blueprint/tutorial.md [hunk 7]
```diff
@@ -331,12 +331,35 @@ You can always modify Blueprints, add/remove properties or change existing prope
...
+
+#### PUT request
+
+Make an HTTP PUT request to the URL `https://api.getport.io/v1/blueprints/{old_blueprint_identifier}` with the Blueprint definition JSON, after updating the value of the `identifier` key in the request body to the new identifier.
```
Comment (EllaFurman) [inline docs/software-catalog/blueprint/tutorial.md:352]: I don't understand if first there needs to be a HTTP request or a change in the identifier key. WHatever is first, should appear first in the sentence.

File: docs/software-catalog/entity/tutorial.md [hunk 8]
```diff
@@ -242,6 +242,30 @@ To Edit a specific property, for example: `version`, send a PATCH request with t
 properties': {"version": "2.29"}
 ```
 
+### Update Entity identifier
+
+In order to change the identifier of an Entity you can use one of the following options:
```
Comment (EllaFurman) [inline docs/software-catalog/entity/tutorial.md:247]:
```suggestion
There are two methods to to change an Entity's identifier:
```

File: docs/software-catalog/entity/tutorial.md [hunk 9]
```diff
@@ -242,6 +242,30 @@ To Edit a specific property, for example: `version`, send a PATCH request with t
...
+
+#### From the UI
+
+1. In either the matching Blueprint page or [Entity page](./entity.md#entity-page), click the 3 dots (`...`) button and then click the `Edit` button;
+2. Update the value of the `identifier` key to the new identifier;
+3. Click `Update`.
+
+#### PUT request
+
+Make an HTTP PUT request to the URL `https://api.getport.io/v1/blueprints/{blueprint_identifier}/entities/{old_entity_identifier}` with the Entity definition JSON, after updating the value of the `identifier` key in the request body to the new identifier.
```
Comment (EllaFurman) [inline docs/software-catalog/entity/tutorial.md:257]: Same problem with the sentence as above.

File: docs/software-catalog/search-in-port.md [hunk 10]
```diff
@@ -132,11 +132,16 @@ The following rule will return Entities whose identifier is `port-api`:
 ```
 
 :::info Available properties
-We can search over a variety of properties:
+You can search over a variety of properties:
```
Comment (EllaFurman) [inline docs/software-catalog/search-in-port.md:135]: what does "search over" mean? is it a tech term?

File: docs/software-catalog/search-in-port.md [hunk 11]
```diff
@@ -132,11 +132,16 @@ The following rule will return Entities whose identifier is `port-api`:
...
 
 - "[Meta-properties](../software-catalog/blueprint/mirror-properties.md#meta-property-mirror-property)" such as `$identifier`, `$title`, `$createdAt` and more.
 - User-defined properties that appear under the `properties` key in the `blueprint` definition
 
+We are currently working on search support for the following property types:
```
Comment (EllaFurman) [inline docs/software-catalog/search-in-port.md:140]:
```suggestion
We are currently working on navigation support for the following property types:
```

**Review by EllaFurman — DISMISSED** [review]

File: docs/software-catalog/search-in-port.md (hunk 10 above)
Comment (MPTG94) [reply docs/software-catalog/search-in-port.md:135]: It means applying a search query over one of the available properties

**Pull request merged**